python/ray/autoscaler/_private/load_metrics.py

```diff
@@ -97,6 +97,7 @@ def update(
         infeasible_bundles: List[Dict[str, float]] = None,
         pending_placement_groups: List[PlacementGroupTableData] = None,
         cluster_full_of_actors_detected: bool = False,
+        node_last_used_time_s: Optional[float] = None,
```

Comment:
In which case will this not be set and the default value be used? How about making the field required instead of optional?

Reply:
In the testing code, manually [call load_metrics.update()](https://github.com/ray-project/ray/blob/58425b9ff03989d5a50cdfb59ac66742b512dab6/python/ray/tests/test_autoscaler.py#L1782).
It's more convenient to set last_used_time default as now.

Reply:
How about making the field required instead? The current implementation makes `ray_nodes_last_used_time_by_ip` have two different definitions.